.gitattributes

```diff
@@ -1,3 +1,4 @@
 *.tar.gz filter=lfs diff=lfs merge=lfs -text
 *.pdf filter=lfs diff=lfs merge=lfs -text
 *.txt filter=lfs diff=lfs merge=lfs -text
+*.gz filter=lfs diff=lfs merge=lfs -text
```

Comment:
Doesn't this line (\*.gz) shadow the first line (\*.tar.gz) ?  We can drop the first line I think.